components/consumers/dependency-track/main.go

```diff
@@ -47,11 +53,40 @@ func main() {
 	if projectUUID == "" {
 		log.Fatal("project uuid is mandatory for dependency track")
 	}
-	c, err := dtrack.NewClient(authURL, dtrack.WithAPIKey(apiKey))
+	if authURL == "" {
+		log.Fatal("auth url is mandatory for dependency track")
+	}
+	if apiKey == "" {
+		log.Fatal("api key is mandatory for dependency track")
+	}
+	if projectName == "" {
+		log.Fatal("project name is mandatory for dependency track")
+	}
+	if projectVersion == "" {
+		log.Fatal("project version is mandatory for dependency track")
+	}
+
+	client, err := dtrack.NewClient(
+		authURL,
+		dtrack.WithHttpClient(
+			&http.Client{Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: debug,
+				},
+			},
+			}),
+		dtrack.WithDebug(debug),
+		dtrack.WithAPIKey(apiKey),
+	)
 	if err != nil {
 		log.Panicf("could not instantiate client err: %#v\n", err)
 	}
-	client = c
+
+	abt, err := client.Metrics.LatestPortfolioMetrics(context.Background())
+	if err != nil {
+		log.Fatalf("cannot connect to Dependency Track at %s, err:'%v'", authURL, err)
+	}
+	slog.Info(fmt.Sprintf("Connection to DT successful, projects in instance: %d", abt.Projects))
```

Comment:
you shouldn't use `Sprintf` with slog, there should be a separate field called `projects`